benchmark/src/main.rs

```diff
@@ -0,0 +1,335 @@
+mod cli;
+mod error;
+mod harness;
+mod io;
+mod ir_utils;
+mod logger;
+mod runner;
+mod stats;
+use crate::cli::*;
+use crate::error::HarvestResult;
+use crate::harness::*;
+use crate::io::*;
+use crate::ir_utils::{cargo_build_result, raw_cargo_package, raw_source};
+use crate::logger::TeeLogger;
+use crate::stats::*;
+use clap::Parser;
+use harvest_ir::HarvestIR;
+use harvest_translate::cli::initialize;
+use harvest_translate::transpile;
+use std::fs::File;
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+
+/// Encapsulate important results from transpilation
+pub struct TranspilationResult {
```

Comment:
It seems like everything that `TranspilationResult` does/contains should instead be handled by a type in `harvest_translate` (i.e. `TranspileOutcome` from #51?).